src/test/java/LoginTests.java

```diff
@@ -0,0 +1,54 @@
+
+import Pages.LoginPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class LoginTests extends BaseTests {
+    @Test
+    void loginSuccessTest() {
+        //1. Login Success.
+
+        LoginPage loginPage = new LoginPage(driver);
```

Comment:
You can create an instance of LoginPage once and then use it for all tests. You should just extract LoginPage from test method and move LoginPage to separate variable